src/stop_motion_obj.py

```diff
@@ -78,7 +79,16 @@ def updateFrame(scene):
     global loadingSequenceLock
     if loadingSequenceLock is False:
         scn = bpy.context.scene
-        setFrameNumber(scn.frame_current)
+        setFrameNumber(scn.frame_current, False)
+
+#Workaround for a bug where Blender crashes when Depsgraph_Update_Pre calls updateFrame and a Single Mesh is used.
```

Comment:
Another minor style thing, but let's put a space between the `#` and the comment, like this: `# Workaround...`